cluster/ephemeral-provider-common.sh

```diff
@@ -46,19 +46,6 @@ function build() {
     ${KUBEVIRT_PATH}hack/dockerized "DOCKER_PREFIX=${DOCKER_PREFIX} DOCKER_TAG=${DOCKER_TAG} KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER} IMAGE_PULL_POLICY=${IMAGE_PULL_POLICY} VERBOSITY=${VERBOSITY} ./hack/build-manifests.sh"
     ${KUBEVIRT_PATH}hack/dockerized "DOCKER_PREFIX=${DOCKER_PREFIX} DOCKER_TAG=${DOCKER_TAG} KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER} ./hack/bazel-build.sh"
     ${KUBEVIRT_PATH}hack/dockerized "DOCKER_PREFIX=${DOCKER_PREFIX} DOCKER_TAG=${DOCKER_TAG} DOCKER_TAG_ALT=${DOCKER_TAG_ALT} KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER} ./hack/bazel-push-images.sh"
-
-    # Make sure that all nodes use the newest images
```

Comment:
i thought we still need this?

Reply:
To be honest, I do not sure, I remember we had some issues with it, but if we had `pullPolicy: Always` for the dev environment, it should get new images on each `make cluster-sync`, also now with podman and cri-o you cannot be sure that the node will have docker at all.
@rmohr Do you remember why do we have it?

Reply:
I believe it was so we'd have the images pre-cached on the nodes.  If CI passes with this removed, then i doubt it's necessary anymore.

Reply:
I can see jobs run without any troubles.

Reply:
It is needed for subsequent cluster-sync runs. Now with this change merged it only works on the first sync.